capitalize_decorator.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'nameable'
+
+class CapitalizeDecorator < Nameable
+  def initialize(nameable)
+    super()
+    @nameable = nameable
+  end
+
+  def correct_name
+    @nameable.correct_name.capitalize
+  end
```

Comment:
- [ ] You have created CapitalizeDecorator 👏 Well done. However, Capitalize Decorator should inherit from a Base Decorator that you have not created yet. Please create a base decorator which inherits from Nameable according to project specifications 👍